src/t8_schemes/t8_default/t8_default_tri/t8_dtri_bits.c

```diff
@@ -1400,6 +1369,9 @@ t8_dtri_init_linear_id_with_level (t8_dtri_t * t, t8_linearidx_t id,
   t8_dtri_type_t      type;
   T8_ASSERT (0 <= id
              && id <= ((t8_linearidx_t) 1) << (T8_DTRI_DIM * end_level));
+  /*TODO: Ensure, that the function is called with a valid element */
```

Comment:
Is there a reason that this is a TODO comment?

Reply:
Deleted the Todo. Done